src/send/EnterAmount.tsx

```diff
@@ -142,21 +133,24 @@ export default function EnterAmount({
     },
   })
 
-  useEffect(() => {
-    onClearPreparedTransactions()
+  useEffect(
+    function refreshPreparedTransactions() {
+      onClearPreparedTransactions()
 
-    const canRefresh =
-      processedAmounts.token.bignum &&
-      processedAmounts.token.bignum.gt(0) &&
-      processedAmounts.token.bignum.lte(token.balance)
-    if (!canRefresh) return
+      const canRefresh =
+        processedAmounts.token.bignum &&
+        processedAmounts.token.bignum.gt(0) &&
+        processedAmounts.token.bignum.lte(token.balance)
+      if (!canRefresh) return
 
-    const debouncedRefreshTransactions = setTimeout(() => {
-      return onRefreshPreparedTransactions(processedAmounts.token.bignum!, token, feeCurrencies)
-    }, FETCH_UPDATED_TRANSACTIONS_DEBOUNCE_TIME_MS)
+      const debouncedRefreshTransactions = setTimeout(() => {
+        return onRefreshPreparedTransactions(processedAmounts.token.bignum!, token, feeCurrencies)
+      }, FETCH_UPDATED_TRANSACTIONS_DEBOUNCE_TIME_MS)
 
-    return () => clearTimeout(debouncedRefreshTransactions)
-  }, [processedAmounts.token.bignum?.toString(), token])
+      return () => clearTimeout(debouncedRefreshTransactions)
+    },
+    [processedAmounts.token.bignum?.toString(), token]
+  )
```

Comment:
Just changed this to be a named function instead of an array function, no other changes to the implementation of this function were introduced.